### PR TITLE
DD-28 API system information

### PR DESCRIPTION
**Pull request by armshare**

### CODE REVIEW

File: DevDynamo.Web/Areas/ApiV1/Controllers/SystemController.cs [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    [Route("api/v1/[controller]")]
+    [ApiController]
+    public class SystemController : AppControllerBase
+    {
+        [HttpGet]
+        public ActionResult<SystemResponse> GetVersionSystem()
+        {
+         
+            var v = GetType().Assembly.GetName().Version.ToString();
+            var envi = Environment.GetEnvironmentVariable("ASPNETCORE_ENVIRONMENT");
```
Comment (surrealist) [inline DevDynamo.Web/Areas/ApiV1/Controllers/SystemController.cs]: We should use the `EnvironmentName` of the `IWebHostEnvironment` injected into the controller. 

Access to the environment variable might not be correct in every case.

File: DevDynamo.Web/Areas/ApiV1/Controllers/SystemController.cs [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+            DateTime utcTime = DateTime.UtcNow;
+            string localTime = utcTime.ToLocalTime().ToString("zzz");
+            string now = utcTime.ToString($"yyyy-MM-dd HH:mm:ss{localTime}");
```
Comment (surrealist) [inline DevDynamo.Web/Areas/ApiV1/Controllers/SystemController.cs]: combine UTC date/time with the timezone of the local. I don't think it's correct. 🤔

File: DevDynamo.Web/Areas/ApiV1/Models/SystemResponse.cs [hunk 3]
```diff
@@ -0,0 +1,13 @@
+﻿using DevDynamo.Models;
+using System.ComponentModel.DataAnnotations;
+
+namespace DevDynamo.Web.Areas.ApiV1.Models
+{
+  public class SystemResponse
+  {
+    public string? version { get; set; }
+    public string? environment { get; set; }
+    public string? now { get; set; }
```
Comment (surrealist) [inline DevDynamo.Web/Areas/ApiV1/Models/SystemResponse.cs]: We should name properties with CamelCase as it is a C# code. When it has returned from the API, it'll be converted to camelCase automatically.

**Review by surrealist — CHANGES_REQUESTED** [review]

**Comment by surrealist** [comment]
Why this PR has so many commits? You might need an updated upstream version.

File: DevDynamo.Web/Areas/ApiV1/Controllers/SystemController.cs (hunk 2 above)
Comment (armshare) [reply DevDynamo.Web/Areas/ApiV1/Controllers/SystemController.cs]: ผมลองใช้ ทั้ง 
```
string utcTime = DateTime.UtcNow.ToString($"yyyy-MM-dd HH:mmzzz"); //2023-07-06 15:32+00:00
string localTime = DateTime.Now.ToString($"yyyy-MM-dd HH:mmzzz"); //2023-07-06 22:32+07:00
```
หรือมันมีมากกว่านี้อ่ะครับ

**Comment by surrealist** [comment]
Thanks for the update. Let's talk tmr.

**Review by surrealist — APPROVED** [review]

**Pull request merged**